requirements.txt

```diff
@@ -1,10 +1,10 @@
 # Everything needed in production
-werkzeug<2.3
+werkzeug>=3.0.3
 Flask>=2.0
 Flask-Bootstrap
 Flask-CORS
 path.py
-Flask-Markdown
+Flask-Misaka
```

Comment:
🇯🇵

Reply:
It looked markupish enough I think:
![Screenshot 2024-05-07 at 09 13 03](https://github.com/Clinical-Genomics/scout/assets/758570/0cbd190a-c102-49f7-a8e7-3d81e824ace4)

Reply:
And so does Python-Markdown with a simple jinja filter wrap:
<img width="805" alt="Screenshot 2024-05-07 at 15 18 57" src="https://github.com/Clinical-Genomics/scout/assets/758570/a30e4db1-f0a5-42eb-a73b-04b4b10dce7e">